drupal/scripts/drush/aliases.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export PATH=/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin
```

Comment:
Would it make more sense to keep the old PATH as well or is this not important?

export PATH=/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:$PATH